install.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+DIRECTORY="fcc-python-vagrant"
+
+echo "This script requires superuser access to install apt packages."
+echo "You will be prompted for your password by sudo."
+
+# clear any previous sudo permission
+sudo -k
+
+# run inside sudo
+sudo sh <<SCRIPT
+
+echo "Installing VirtualBox ..."
```

Comment:
This works! But it would be even more robust, if you would detect lower version and uninstall them, and reinstall them.